app/lib/azureService.js

```diff
@@ -0,0 +1,48 @@
+const azure = require('azure-storage');
+
+const config = require('./config.js');
+
+const blobSvc = azure.createBlobService();
+
+function listBlobs() {
+  return new Promise((resolve, reject) => {
+    blobSvc.listBlobsSegmented(config.containerName, null, (error, result) => {
+      if (!error) {
+        // result.entries contains the entries
+        // If not all blobs were returned, result.continuationToken has the continuation token.
```

Comment:
This comment doesn't make any sense to me. I can see `result.entries` is returned from the function so I'd expect that to contain the blobs. And I don't see how knowing `result.continuationToken` contains the continuation token helps. Perhaps it would if something was done with.